views/index.ejs

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= title %></title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+    <h1>Welcome to <%= title %>!</h1>
+    <p>The site that lets you replace sensitive text of your choosing with the letter X.</p><br>
+    <p>In the box below, enter the words you wish to be redacted, separated by a space:</p>
+    <form method="post">
```

Comment:
In future iterations, I need to handle the texts that's submitted upon hitting the submit button, likely save it to a database, transform it and then re-route the transformed text to the view.